File: app/src/main/java/xyz/hasnat/csdictionary/WelcomeSlider/WelcomeActivity.java
package xyz.hasnat.csdictionary.WelcomeSlider;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import xyz.hasnat.csdictionary.R;

public class WelcomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        startActivity(new Intent(WelcomeActivity.this, IntroActivity.class));
        finish();
        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);

    }
}

File: app/src/main/java/xyz/hasnat/csdictionary/intro_slider/WelcomeActivity.kt
package xyz.hasnat.csdictionary.intro_slider

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class WelcomeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        startActivity(Intent(this@WelcomeActivity, IntroActivity::class.java))
        finish()
        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)

    }
}
